src/main/resources/archetype-resources/silo/src/main/java/silo/configuration/PersistenceConfiguration.java

```diff
@@ -3,25 +3,12 @@
 #set($symbol_escape='\' )
 package ${package}.${artifactId}.configuration;
 
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.Profile;
-import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseBuilder;
-import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseType;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
 
 import javax.sql.DataSource;
 
-@Configuration
-@Profile("!testing")
+@EnableJpaRepositories(basePackages = "${package}.${artifactId}")
```

Comment:
You already have that annotated here: https://github.com/rebuy-de/rebuy-silo-archetype/pull/14/files#diff-3405101acf837027c5898c19b16037fcR19

I don't know if this configuration does anything after removing the annotation. So this class may become obsolete.